scripts/fetch_and_build_deb

```diff
@@ -176,14 +176,13 @@ mjobs="$(expr $procs + 1)"
 DEB_BUILD_OPTIONS="parallel=${mjobs}" debuild --prepend-path /usr/local/bin --preserve-envvar CONF_EXTRA_VERSION --preserve-envvar UNENCRYPTED_PACKAGE --preserve-envvar PACKAGE_ENCRYPTION_KEY -uc -us -B --lintian-opts --profile debian --allow-root
 
 ## Rename all *.ddeb files to *.deb since we upload debug packages with .deb suffix
-for filename in ../*.ddeb
-do
+for file in ../*.ddeb; do
   # .ddeb suffix is automatically chosen when naming debug packages for Ubuntu
   # releases. For this reason, we rename all *.ddeb files to *.deb since we upload
   # debug packages with .deb suffix.
-  if [ ${filename} != "../*.ddeb" ]; then
+  if [ "${file}" != "../*.ddeb" ]; then
```

Comment:
I still do not get why we need this checks here. Do you know how the `file` variable can store the string `../*.ddeb` ?

Reply:
https://stackoverflow.com/a/6364244/13370386

Reply:
if ddeb file does not exist, filename becomes ../*.ddeb

Reply:
```suggestion

  if [ -e "${file}" ]; then
```

I think this makes it easier to understand what is going on.

Reply:
Fixed